front/components/assistant/details/tabs/AgentTriggersTab.tsx

```diff
@@ -81,43 +108,14 @@ export function AgentTriggersTab({
                     <div className="font-semibold">{trigger.name}</div>
                   </div>
                   <div className="self-end">
-                    {trigger.isEditor ? (
-                      <Button
-                        label="Manage"
-                        icon={PencilSquareIcon}
-                        href={editionURL}
-                        variant="outline"
-                        size="sm"
-                      />
-                    ) : trigger.isSubscriber ? (
-                      <Button
-                        label="Unsubscribe"
-                        icon={XMarkIcon}
-                        variant="outline"
-                        size="sm"
-                        isLoading={isLoading}
-                        disabled={isLoading}
-                        onClick={async () => {
-                          setIsLoading(true);
-                          await unsubscribe(trigger.sId);
-                          setIsLoading(false);
-                        }}
-                      />
-                    ) : (
-                      <Button
-                        label="Subscribe"
-                        icon={ArrowRightIcon}
-                        variant="outline"
-                        size="sm"
-                        isLoading={isLoading}
-                        disabled={isLoading}
-                        onClick={async () => {
-                          setIsLoading(true);
-                          await subscribe(trigger.sId);
-                          setIsLoading(false);
-                        }}
-                      />
-                    )}
+                    <Button
+                      label="Delete"
+                      disabled={!trigger.isEditor}
```

Comment:
I understand why it's here but on our case should never happen right? 
Cause we don't want other user's triggers on Dust accessible here 😄